filebeat/module/pensando/dfw/_meta/fields.yml

```diff
@@ -0,0 +1,59 @@
+- name: payload_raw
+  description: Please add description
```

Comment:
This `payload_raw` get removed in the ingest node pipeline from what I see so this field can be removed.

Reply:
Fixed this in 0a619b9c670384d33954d65a8f3ad0ee7254c8af